x/evm/keeper/keeper.go

```diff
@@ -62,6 +62,8 @@ type Keeper struct {
 	// fetch EIP1559 base fee and parameters
 	feeMarketKeeper types.FeeMarketKeeper
 
+	ibcKeeper *ibckeeper.Keeper
```

Comment:
this seems like a design smell, the ibc precompile should not need this at evm keeper?